content/en/docs/setup/best-practices/certificates.md

```diff
@@ -57,7 +58,18 @@ Required CAs:
 | front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
 
 On top of the above CAs, it is also necessary to get a public/private key pair for service account management, `sa.key` and `sa.pub`.
+The following example illustrates the CA key and certificate files shown in the previous table:
 
+```
+/etc/kubernetes/pki/
+├── ca.crt
+├── ca.key
+├── etcd
+│   ├── ca.crt
+│   ├── ca.key
+├── front-proxy-ca.crt
+└── front-proxy-ca.key
```

Comment:
How about representing this more like the output from `find`:
```suggestion
/etc/kubernetes/pki/
/etc/kubernetes/pki/ca.crt
/etc/kubernetes/pki/ca.key
/etc/kubernetes/pki/front-proxy-ca.crt
/etc/kubernetes/pki/front-proxy-ca.key
/etc/kubernetes/pki/etcd/
/etc/kubernetes/pki/etcd/ca.crt
/etc/kubernetes/pki/etcd/ca.key
```
I think most readers could follow that, and there's the additional benefit that they can copy and paste those paths.

Reply:
@sftim how strongly do you feel about this nit? With tech review, I feel like this PR could be (squashed and) merged with follow up iterations if needed.

However, I could be swayed, as I see a nit with this tree layout in line 69 (`│   ├── ca.key` vs `│   └── ca.key`) 😄

Reply:
I'm going to leave LGTM to SIG Auth. The feedback here isn't a veto _but_ I'm not keen on the ASCII art. I think it adds mental effort for the reader to join the paths in their head.

Reply:
I changed the tree view of the files to a regular full path listing (like you might see with ls -l) as you suggested. I listed the files in the order in which they appear in each table and removed the directory listings.